benchmarks/synth-bm/justfile

```diff
@@ -18,7 +18,7 @@ create_sub_accounts:
         --num-sub-accounts 100 \
         --deposit 953060601875000000010000 \
         --channel-buffer-size 1200 \
-        --interval-duration-micros 800 \
+        --requests-per-second 2000 \
```

Comment:
Interval of 800 of corresponds to 1250 RPS, which at some point was close to max for account creation. Did RPS 2000 here work for you against a node? If not (or untested), I'd set RPS to 1250 to avoid breaking the justfile recipe.